docl/install_rpm_server.py

```diff
@@ -61,27 +65,41 @@ def _serve():
         host, port, work.cached_install_rpm_path.basename())
     process = serve(work.dir, host=host, port=port, _bg=True)
     logger.info('Install RPM available at {}'.format(local_rpm_url))
+    _wait_for_file_server(local_rpm_url)
```

Comment:
any idea what was this not required before?

Reply:
oh, probably because by the time the bootstrap got to the part where it starts downloading, it took a while and the fileserver was already up?

Reply:
Yeah, probably. But after playing around with it a bit, this seemed the safest route to ensure it's working.